waiter/config-full.edn

```diff
@@ -1001,6 +1001,14 @@
  ;; Components also get passed in the :kv-store-factory, :leader?-fn and :synchronize-fn for use in the factory.
  :custom-components {:in-memory-kv-store {:factory-fn waiter.kv/new-local-kv-store}}
 
+ ;; Configure event maintainer that handles internal events to be sent externally
+ :event-maintainer-config {
+                           ;; Buffer size of the event-chan where other threads can push internal events to
+                           :event-chan-buffer-size 1000
+
+                           ;; The fn that handles new instance-failure events
+                           :instance-failure-event-handler-fn 'waiter.event-handler/default-instance-failure-event-handler}
```

Comment:
If we listen on `router-state-push-mult` then we do not need a separate `event-chan`
```suggestion
 ;; Configure instance notifier that handles notifications for instances
 :instance-tracker-config {
                           ;; function that triggers notification, it needs access to service-id, service-description, and map of current and delat of different kinds of instances, we can start with current failed instances and delta of failed instances.
                           :notify-service-fn ...}
```